### PR TITLE
Upgrade Hazelcast example with the latest hazelcast-kubernetes-bootst…

### PR DESCRIPTION
**Pull request by oscerd**
…rapper 0.8.0, using Deployments instead of Replication Controller.

**What this PR does / why we need it**:
This PR upgrade the Hazelcast example in storage folder. It would be great to be aligned with the latest version of hazelcast-kubernetes-bootstrapper

### CODE REVIEW

**Comment by oscerd** [comment]
I signed it

**Comment by thockin** [comment]
@pires can you take a look?

**Comment by pires** [comment]
@thockin I will first thing tomorrow.

@oscerd I'm about to release a new version, will ping you here when I release it tomorrow. Thanks for picking this!

File: examples/storage/hazelcast/README.md
Comment (pires) [inline examples/storage/hazelcast/README.md]: "its set"

File: examples/storage/hazelcast/README.md
Comment (pires) [inline examples/storage/hazelcast/README.md]: Where is "1 cpu core" requirement/limit defined?

File: examples/storage/hazelcast/image/Dockerfile
Comment (pires) [inline examples/storage/hazelcast/image/Dockerfile]: Why is the Dockerfile included if the image used here is from my quay.io repo?

**Review by pires — CHANGES_REQUESTED** [review]

**Comment by oscerd** [comment]
@pires @thockin I'll update soon the PR and wait for the next release

**Comment by oscerd** [comment]
PR updated.

**Comment by pires** [comment]
@oscerd version 0.8.0 released and updated instructions.

**Comment by oscerd** [comment]
I will align to 0.8.0

**Comment by oscerd** [comment]
@pires @thockin it should be fine

**Review by pires — APPROVED** [review]
/lgtm

**Comment by pires** [comment]
@thockin need you or someone to remove the release-note-label-needed label.

**Comment by pires** [comment]
Will the bot ignore `release-note-label-needed`? /cc @apelisse

**Comment by thockin** [comment]
@k8s-bot ok to test

**Comment by pires** [comment]
@oscerd seems you have to add/update a test.

**Comment by oscerd** [comment]
I hope now it's fine

**Comment by oscerd** [comment]
We need just the lgtm label

**Comment by oscerd** [comment]
Any news?

**Comment by oscerd** [comment]
Not sure, but the test error doesn't look something related to this PR

**Comment by pires** [comment]
@k8s-bot kops aws e2e test this

**Comment by oscerd** [comment]
It's OK now

**Comment by yujuhong** [comment]
The Hazelcast tests have been consistently failing since this PR was merged: https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-examples/
Could you please check the failures? @oscerd @pires

**Comment by oscerd** [comment]
The hazelcast e2e test is fixed with this PR: https://github.com/kubernetes/kubernetes/pull/41310

**Pull request merged**
